features/unsupported/USBDevice/USBAudio/USBAudio.cpp

```diff
@@ -113,9 +113,12 @@ bool USBAudio::write(uint8_t * buf) {
     return true;
 }
 
-void USBAudio::writeSync(uint8_t *buf)
+void USBAudio::writeSync(uint8_t *buf, int jitter_nb)
 {
-    USBDevice::writeNB(EPISO_IN, buf, PACKET_SIZE_ISO_OUT, PACKET_SIZE_ISO_OUT);
+    if ((jitter_nb != -1) && (jitter_nb !=1)) {
```

Comment:
The 3 possible values are -1,0,1. the comment says  * @param jitter_nb : number of sample to add or remove.  Is following comment better * @param jitter_nb : -1 (remove one sample), 1(add one sample), 0(no sample removal/add).
or you prefer a meaningfull enum ?